layouts/shortcodes/ct_logs.html

```diff
@@ -0,0 +1,10 @@
+
+{{ range $.Site.Data.transparency.transparencyLogs }}
+<p>
+  <strong>{{ i18n "ct_name" }}: {{ .name }}</strong><br>
```

Comment:
In some languages (e.g. French, colon ":" must be prefixed by a space:
```suggestion
  <strong>{{ i18n "ct_name" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .name }}</strong><br>
```
Same for all ":" of the file

Reply:
Done.
Although, it might be easier to just move `:` into the translated strings because some other languages might have any other punctuation conventions. In particular, I'm not sure how right-to-left languages will behave. I think this should work fine, but I'm not sure.